src/lib/rest/httpRequestSend.cpp

```diff
@@ -630,7 +655,7 @@ std::string sendHttpSocket
   /* We add a final newline (I guess that HTTP protocol needs it) */
   strcat(msg, "\n");
 
-  int fd = socketHttpConnect(ip, port); // Connecting to HTTP server
+  int fd = (ip, port); // Connecting to HTTP server
```

Comment:
Is this syntactically correct C++ code? Maybe the function name replacement fails...